dataDownloader/unicodecsv.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, f, dialect=csv.excel, encoding="utf-8", **kwds):
         self.encoder = codecs.getincrementalencoder(encoding)()
 
     def writerow(self, row):
-        self.writer.writerow([s.encode("utf-8") for s in row])
+        self.writer.writerow([s for s in row])
```

Comment:
en ese caso no es necesario el for, debería quedar así: `self.writer.writerow(row)`

Reply:
Corregido.

Reply:
esto está cubierto por algún test? si es así se podría agregar una validación en la salida para darnos cuentas si vuelve a pasar en el futuro. Si no está cubierto ves una forma de escribir un test que cubra esta situación?

Reply:
Ese módulo completo no está siendo testeadol, podría aprovechar de hacerlo si no hay cosas más prioritarias.

Reply:
ok, si hagamos eso 👍